internal/runner/runner.go

```diff
@@ -76,6 +94,10 @@ func (r *Runner) Run() error {
 		}
 	}
 	for _, tool := range r.options.Update {
+		if !path.IsSubPath(homeDir, r.options.Path) {
+			gologger.Error().Msgf("skipping update outside home folder: %s", tool)
+			continue
+		}
```

Comment:
I think logic here is different `CRUD operations for tools within $home directory`
@Mzack9999  shouldn't pdtm check path of `nuclei(or other tool)` when updating ?

Reply:
I think here we are doing exactly this, if the provided path is not within the home folder, we skip update. Same happens for install